erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -441,6 +441,17 @@ int WebRtcConnection::deliverFeedback_(std::shared_ptr<DataPacket> fb_packet) {
   return fb_packet->length;
 }
 
+int WebRtcConnection::deliverEvent_(MediaEventPtr event) {
+  if (!sending_ || getCurrentState() != CONN_READY) {
```

Comment:
I'm wondering if it would be useful to have events that are propagated to the pipeline even before the connection is ready

Reply:
yep, makes sense